### PR TITLE
mappings: update job mapping

### PR DESCRIPTION
**Pull request by ammirate**
* INSPIR-2419

Signed-off-by: Antonio Cesarano <cesarano2607@gmail.comk>

### CODE REVIEW

File: inspirehep/search/mappings/v5/records/jobs.json [hunk 1]
```diff
@@ -8,57 +8,100 @@
...
+              },
+              "type": "object"
+            }
+          },
+          "type": "object"
+        },
+        "acquisition_source": {
+          "properties": {
+            "$ref": {
+              "type": "keyword"
```
Comment (michamos) [inline inspirehep/search/mappings/v5/records/jobs.json]: I think you got confused between `$ref` to reference other schema elements and `$ref` inside the fields to reference other records.

File: inspirehep/search/mappings/v5/records/jobs.json [hunk 2]
```diff
@@ -8,57 +8,100 @@
...
+            }
+          },
+          "type": "object"
+        },
+        "arxiv_categories": {
+          "properties": {
+            "$ref": {
+              "type": "keyword"
```
Comment (michamos) [inline inspirehep/search/mappings/v5/records/jobs.json]: same

**Comment by michamos** [comment]
don't you also want to bump `inspire-schemas` or will this be done in another PR?

**Comment by ammirate** [comment]
The bump will be done separately, since there's the whole chain of dependencies to update:
* inspire-utils
* inspire-matcher
* inspire-dojson
* inspire-json-merger
and something else probably.

**Comment by harunurhan** [comment]
What is blocking this one? @ammirate 

I need it for the Jobs search... well also `inspire-schemas` bump (@monaawi would need new schema too)

**Comment by ammirate** [comment]
The comment above is listing the dependencies for this. 
I'm merging it, since we can bump all of them now.

**Pull request merged**